docs/dev/agent_integration.md

```diff
@@ -332,79 +398,11 @@ To generate `conf.yaml.example` using `ddev`, run:
 ddev validate config --sync awesome
 ```
 
-{{% /tab %}}
-{{% tab "Manifest" %}}
-
-For this example, the `awesome/manifest.json` for the Awesome Service Check appears in the following format:
-```json
-{
-  "manifest_version": "2.0.0",
-  "app_uuid": "79eb6e54-2110-4d50-86c3-f7037d1a9daa", // Do not use this example UUID. UUIDs must be unique and valid.
-  "app_id": "awesome",
-  "classifier_tags": [
-    "Supported OS::Linux",
-    "Supported OS::macOS",
-    "Supported OS::Windows"
-  ],
-  "display_on_public_website": false,
-  "tile": {
-    "overview": "README.md#Overview",
-    "configuration": "README.md#Setup",
-    "support": "README.md#Support",
-    "changelog": "CHANGELOG.md",
-    "description": "",
-    "title": "Awesome",
-    "media": []
-  },
-  "author": {
-    "support_email": "email@example.org"
-  },
-  "oauth": {},
-  "assets": {
-    "integration": {
-      "source_type_name": "Awesome",
-      "configuration": {
-        "spec": "assets/configuration/spec.yaml"
-      },
-      "events": {
-        "creates_events": false
-      },
-      "metrics": {
-        "prefix": "awesome.",
-        "check": "",
-        "metadata_path": "metadata.csv"
-      },
-      "service_checks": {
-        "metadata_path": "assets/service_checks.json"
-      }
-    }
-  }
-}
-```
-
 {{% /tab %}}
 {{% tab "Metadata" %}}
 
 For this example, the Awesome integration doesn't provide any metrics, so in this case, the generated `awesome/metadata.csv` only contains only a row with the column names.
```

Comment:
I find this tab confusing. It looks to be blank, and doesn't show the row with column names.

Reply:
Just calling it "metadata" and not "metadata.csv" also makes it hard to follow. 

More than that, though, is there a reason why we are showing this "metadata" example next to "Configuration template"? There aren't examples for readme, the manifest, etc.

Reply:
@hestonhoffman I think Alex's feedback above falls into your wheelhouse - can you provide context on why we only show previews for a few of the asset types? Perhaps we don't need to show previews at all?